include/igl/serialize.h

```diff
@@ -105,21 +106,21 @@ namespace igl
   //   obj        object to load back serialization to
   //
   template <typename T>
-  inline bool deserialize(T& obj,const std::string& filename);
+  inline bool deserialize(T& obj,const std::string& filename,const bool warning = false);
```

Comment:
I thought we should default to having `warning = true`? Also, no need to make it `const bool` (`bool` is enough).